app/routes/docs/about/history.mdx

```diff
@@ -0,0 +1,48 @@
+---
+handle:
+  breadcrumb: History
+---
+
+import history from '~/img/gcn_history.png'
+
+# History of GCN
+
+The development of GCN began in 1992 as an innovative solution to a major hardware failure aboard the Compton Gamma-ray Observatory ([CGRO](https://heasarc.gsfc.nasa.gov/docs/cgro/cgro.html), 1991-2000). The mission was designed such that data was stored on a spacecraft tape recorder and downloaded several times per day via scheduled contacts with NASA's Tracking and Data Relay ([TDRS](https://www.nasa.gov/directorates/heo/scan/services/networks/tdrs_main/)) satellites, which then relayed the data to a ground station. Less than two years into the CGRO mission the tape recorder failed, threatening the continuation of this Great Observatory. To save CGRO, NASA arranged for continuous, real-time transmission of the science data through the TDRS system.
+
+## BACODINE
+
+Dr. Scott Barthelmy at NASA/GSFC realized that this new data mode created an opportunity -- near-time notification of gamma-ray bursts detected by CGRO's Burst And Transient Search Experiment ([BATSE](https://heasarc.gsfc.nasa.gov/docs/cgro/batse/)) instrument. He went on to launch the BAtse COordinates DIstribution NEtwork (BACODINE) system in 1993. Software monitored the real-time telemetry of BATSE and produced an automatic localization within seconds, with a later improvement of a more accurate localization produced by BATSE scientists, after a delay of tens of minutes. The BATSE scientist were notified by BACODINE via pager when a transient occurred. They localized the source and sent the information to the BACODINE system for distribution.
+
+The [GCN Notices](/notices) distribution mechanisms expanded over the following years: phone connections (July 1993), e-mail (December 1993), direct computer-to-computer socket connections over the Internet (January 1994), and by alpha-numeric pagers (September 1994) to any interested follow-up observatories within seconds of detection.
+
+<img
+  src={history}
+  width="900"
+  height="250"
+  loading="lazy"
+  alt="Timeline of GCN history"
+/>
+
+The first robotic follow-up instrument to connect to BACODINE was the Gamma-Ray Optical Counterpart Search Experiments ([GROCSE](https://ui.adsabs.harvard.edu/abs/1995Ap%26SS.231..255A/abstract)) in August 1993. Additional capabilities were added to BACODINE throughout the mid 1990's including trigger classification (GRBs from non-GRBs), monitoring (including 24/7 staffing) and the first incarnation of the interplanetary network ([IPN](https://heasarc.gsfc.nasa.gov/docs/heasarc/missions/ipn.html)).
```

Comment:
This should be maybe "the modern incarnation" or "the third incarnation" of the InterPlanetary Network. The first was ~1978-1983. Separation between second and third is a bit fuzzy, but I think we were in the 3rd by 1993